corporate/lib/stripe.py

```diff
@@ -489,6 +489,39 @@ def __init__(self, event_type: AuditLogEventType) -> None:
         super().__init__(self.message)
 
 
+class UpgradePageParams(TypedDict):
+    annual_price: int
+    demo_organization_scheduled_deletion_date: Optional[datetime]
+    monthly_price: int
+    seat_count: int
+
+
+class UpgradePageCustomerSpecificContext(TypedDict):
+    customer_name: str
+    email: str
+    is_demo_organization: bool
+    demo_organization_scheduled_deletion_date: Optional[datetime]
+    is_self_hosting: bool
```

Comment:
A bunch of the other fields in `UpgradePageContext` are arguably customer specific; I suppose we could call this `UpgradePageSessionTypeSpecificContext` to reference this being stuff that varies with the "session type".

Reply:
Changed.